tutorial_submission_policy.md

```diff
@@ -0,0 +1,64 @@
+# PyTorch Tutorial Submission Policy
+
+This policy outlines the criteria and process for submitting new tutorials to the PyTorch community.
+Our goal is to ensure that all tutorials are of high quality, relevant, and up-to-date, supporting
+both the growth of our users and the evolution of the PyTorch framework itself. By following these
+guidelines, contributors can help us maintain a robust and informative educational environment.
+
+## Acceptance Criteria For New Tutorials
+We are committed to enhancing the learning experience for PyTorch users by providing high-quality
+tutorials. We accept new tutorials that adhere to one of the following use cases:
+
+* **Support for New PyTorch Features:** Tutorials that support new features for upcoming PyTorch
+releases are typically authored by the engineers who are developing these features. These tutorials
+are crucial for showcasing the latest advancements in PyTorch.
+
+* **Tutorials from PyTorch Partners:** We welcome tutorials from our partners who are actively
```

Comment:
I'm not sure why a partner gets preferential treatment it seems like? Most important IMHO is for the content to be interesting and informative